odoo/addons/base/models/res_bank.py

```diff
@@ -106,7 +107,11 @@ def retrieve_acc_type(self, acc_number):
         """ To be overridden by subclasses in order to support other account_types.
         """
         return 'bank'
-    
+
+    def _valid_field_parameter(self, field, name):
+        # Y though ?
+        return name == 'tracking' or super()._valid_field_parameter(field, name)
```

Comment:
Because you're adding `tracking=True` fields to `base`, and the attribute is defined in `mail`.

So don't do that, override the fields in account or whatever.